### DISCUSSION TITLE
E se Tim Burton dirigisse Pokémon?

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#375] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/e-se-tim-burton-dirigisse-pokemon/

**Comment by brunoaugusto** [comment]
O Machamp ficou aterrorizante! :o

**Comment by one-eyed-willy** [comment]
O trabalho do artista é tão impressionante que eu poderia jurar que já vi pelo menos meia dúzia desses em uns desenhos animados por aí

**Comment by angellisy** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 21 de março de 2022](<https://www.vish.blog.br/imagens/e-se-tim-burton-dirigisse-pokemon/#comment_1074139710>):

> O trabalho do artista é tão impressionante que eu poderia jurar que já vi pelo menos meia dúzia desses em uns desenhos animados por aí

Não é um Efeito Mandela, não?

**Comment by one-eyed-willy** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 21 de março de 2022](<https://www.vish.blog.br/imagens/e-se-tim-burton-dirigisse-pokemon/#comment_1074140587>):

> Não é um Efeito Mandela, não?

Efeito Mandela?

**Comment by najarin-the-loremaster** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 21 de março de 2022](<https://www.vish.blog.br/imagens/e-se-tim-burton-dirigisse-pokemon/#comment_1074141043>):

> Efeito Mandela?

Uma dessas "Teorias da Conspiração" que, confesso, faz muito sentido.



Aqui um ponto de partida pra você: https://super.abril.com.br/mundo-estranho/teoria-da-conspiracao-o-assustador-efeito-mandela/

**Comment by brunoaugusto** [comment]
> [Najarin, the Loremaster](</perfil/?user=najarin-the-loremaster>) disse [segunda-feira, 21 de março de 2022](<https://www.vish.blog.br/imagens/e-se-tim-burton-dirigisse-pokemon/#comment_1074156220>):

> Uma dessas "Teorias da Conspiração" que, confesso, faz muito sentido.Aqui um ponto de partida pra você: [https://super.abril.com.br/mundo-estranho/teoria-da-conspiracao-o-assustador-efeito-mandela/](<https://super.abril.com.br/mundo-estranho/teoria-da-conspiracao-o-assustador-efeito-mandela/>)

COMO ASSIM O FILME DO GÊNIO SHAZAAM NUNCA EXISTIU?! :O



Eu tenho certeza que assisti, tanto esse quanto o tal "Kazaam" do Shaquille O'neal
